modules/apps/transfer/types/trace_test.go

```diff
@@ -94,37 +94,6 @@ func TestDenomTrace_Validate(t *testing.T) {
 	}
 }
 
-func TestValidatePrefixedDenom(t *testing.T) {
```

Comment:
moved relevant test cases to packet data validate test func, but maybe can be moved to the extract test instead